### ISSUE TITLE
`Data_Availability_Type` test often fails on PRs from forked repos

### ISSUE THREAD

**Issue by geoknee** [ethereum-optimism/superchain-registry#560]
See for example here https://app.circleci.com/pipelines/github/ethereum-optimism/superchain-registry/7480/workflows/b88ef5ef-3cd2-44dc-b626-84719e3fc5e9/jobs/23511/parallel-runs/0/steps/0-104

```
 === FAIL: github.com/ethereum-optimism/superchain-registry/validation TestValidation/Lisk_Sepolia_Testnet_(4202)/Data_Availability_Type (0.48s)
    data-availability-type_test.go:52: 
                Error Trace:    /home/circleci/project/validation/data-availability-type_test.go:52
                                                        /home/circleci/project/validation/validation_test.go:42
                Error:          Received unexpected error:
                                failed to retrieve previous nonce: missing trie node ca90aa20f2d249344364c8edde61abc21f97a63dbf6b37f54b757c5cbbe03511 (path ) state 0xca90aa20f2d249344364c8edde61abc21f97a63dbf6b37f54b757c5cbbe03511 is not available
                Test:           TestValidation/Lisk_Sepolia_Testnet_(4202)/Data_Availability_Type
                Messages:       failed when checking chain for recent batcher txs from 0x246E119a5BcC2875161b23E4e602e25cEcE96E37
```

The failure is due to the L1 RPC endpoint not being an archive node. Typically there would only be an archive node if it is a paid-for endpoint; we use such an endpoint in CI for PRs in our own org's repo (sharing via secret env vars). Sharing the env vars to forked repos is a security risk, so we need to think about possible ways forward without doing that.

**Comment by geoknee** [comment]
This should be addressed by #564.

We will want to write a runbook explaining how an engineer needs to green-light the CI checks from PRs from forks. They will need to check there is no malicious code.

**Comment by bitwiseguy** [comment]
This issue is completed by this pr: [use the proxyd endpoints and configure circle ci jobs to use known ip range](https://github.com/ethereum-optimism/superchain-registry/pull/605)
